doc/apiary/v2/fiware-ngsi-simple-cookbook.apib

```diff
@@ -0,0 +1,872 @@
+
++ Parameters
+    + id: DC_S1_D41 (optional, string) - Only retrieve entities with this id
+
++ Response 200 (application/json)
+
+        [
+         {
+            "type": "Room",
+            "id": "DC_S1_D41",
+            "temperature": 35.6
+         }
+        ]
+
+### List entities by list of ids [GET /v2/entities{?id}]
+
++ Parameters
+    + id: DC_S1_D41, Boe_Idearium (optional, string) - Id list with the entities to be retrieved
+
++ Response 200 (application/json)
+
+        [
+         {
+            "type": "Room",
+            "id": "DC_S1_D41",
+            "temperature": 35.6
+         },
+         {
+            "type": "Room",
+            "id": "Boe_Idearium",
+            "temperature": 22.5
+         }
+        ]
+
+### List all entities of an specific type [GET /v2/entities{?type}]
+
++ Parameters
+    + type: Room (optional, string) - Only retrieve entities of this type
+
++ Response 200 (application/json)
+
+        [
+         {
+            "type": "Room",
+            "id": "DC_S1_D41",
+            "temperature": 35.6
+         },
+         {
+            "type": "Room",
+            "id": "Boe_Idearium",
+            "temperature": 22.5
+         }
+        ]
+
+### List all entities by id pattern matching [GET /v2/entities{?idPattern}]
+
++ Parameters
+    + pattern: DC_* (optional, string) - Only retrieve entities which id matches this pattern.
+
++ Response 200 (application/json)
+
+        [
+         {
+            "type": "Room",
+            "id": "DC_S1_D41",
+            "temperature": 35.6
+         }
+        ]
+
+### List all Entities. Pagination [GET /v2/entities{?limit,offset,count}]
+
++ Parameters
+    + count: true
+    + limit: 2
+    + offset: 2
+
++ Response 200 (application/json)
+
+    + Headers
+
+            X-Total-Count: 30
+            Link: http://context.example.org/v2/entities?offset=4&limit=2; rel="next", http://context.example.org/v2/entities?offset=28&limit=2; rel="last", http://context.example.org/v2/entities?offset=0&limit=2; rel="first", http://context.example.org/v2/entities?offset=2&limit=2; rel="prev"
+
+    + Body
+
+            [{
+                "type": "Room",
+                "id": "DC_S1_D41",
+                "temperature": 35.6
+            },
+            {
+                "type": "Room",
+                "id": "Boe_Idearium",
+                "temperature": 22.5
+            }]
+
+## Querying context [/v2/entities{?q,limit,offset,coords,geometry,attrs,count}]
+
+
+### Query Entities [GET /v2/entities{?q,limit,offset,coords,geometry,attrs,count}]
+
++ Parameters
+    + q: temperature=22.5;humidity=60 (optional, string) - General query parameter that allows
+    to filter by attribute content
+    + coords: 41.3763726, 2.1864475 (optional, string) - Coordinates. Comma separated values.
+    The number of values will depend on the 'geometry' attribute.
+    + geometry: point (optional, string) - It can be any type defined by the
```

Comment:
Point is not currently support. I would suggest to base the example in `circle`.